python/src/main/java/org/apache/zeppelin/python/PythonCondaInterpreter.java

```diff
@@ -115,22 +115,33 @@ public InterpreterResult interpret(String st, InterpreterContext context) {
   private void changePythonEnvironment(String envName)
       throws IOException, InterruptedException {
     PythonInterpreter python = getPythonInterpreter();
+    String condaLibraryPath = null;
     String binPath = null;
     if (envName == null) {
       binPath = getProperty(ZEPPELIN_PYTHON);
       if (binPath == null) {
         binPath = DEFAULT_ZEPPELIN_PYTHON;
       }
+      condaLibraryPath = null;
     } else {
       Map<String, String> envList = getCondaEnvs();
+      for (String key : envList.keySet()) {
+      }
       for (String name : envList.keySet()) {
         if (envName.equals(name)) {
           binPath = envList.get(name) + CONDA_PYTHON_PATH;
+          condaLibraryPath = String.format("%s/lib", envList.get(name));
```

Comment:
is this quite standard? is it possible the lib path is not env/lib?

Reply:
Yes, it is guaranteed.
python installed via conda is always guaranteed path.